tests/test_invenio_accounts.py

```diff
@@ -136,6 +136,27 @@ def test_datastore_rolecreate(app):
         assert 1 == Role.query.filter_by(name="superuser").count()
 
 
+def test_datastore_update_role(app):
+    """Test update role."""
+    ds = app.extensions["invenio-accounts"].datastore
+
+    with app.app_context():
```

Comment:
Is the context manager needed, or does the `app` fixture already push an app ctx?
```suggestion

```